style/fontsizes.less

```diff
@@ -0,0 +1,26 @@
+@standard: 14px, 15px, 16px, 17px, 18px, 19px, 20px;
+@title: 22px, 23px, 24px, 25px, 26px, 27px, 28px;
+@preview-title: 15px, 16px, 17px, 18px, 19px, 20px, 21px;
+@desc: 12px, 13px, 14px, 15px, 16px, 17px, 18px;
```

Comment:
Those are all sequential so you could also have 4 different bases and go `base + value`